docs/pages/concepts/evaluation_dimension.md

```diff
@@ -0,0 +1,92 @@
+## Overview
+
+Evaluation dimensions are used to evaluate the quality of social interactions.
```

Comment:
make sure mention they can use `SotopiaDimension` here as well, and people don't need to initialize the database when using that